lms/djangoapps/course_api/fields.py

```diff
@@ -0,0 +1,26 @@
+""" Custom fields for Django REST Framework. """
+
+from __future__ import unicode_literals
+
+from rest_framework import fields
+
+
+class AbsoluteURLField(fields.URLField):
```

Comment:
This is a generically useful class.  Thanks for creating it.  Please consider putting it at a common location: https://github.com/edx/edx-platform/blob/master/openedx/core/lib/api/fields.py